src/input/rtl_tcp.cpp

```diff
@@ -546,8 +546,7 @@ void CRTL_TCP_Client::agcTimer(void)
         }
         else { // AGC is off or unknown tuner
             if (minAmplitude == 0 || maxAmplitude == 255) {
-                std::string text = QT_TRANSLATE_NOOP("CRadioController", "ADC overload."
-                    " Maybe you are using a too high gain.");
+                std::string text = QT_TRANSLATE_NOOP("CRadioController", "ADC overload. Maybe you are using a too high gain.");
```

Comment:
Even if this just seems to undo consistent line breaking (which I'd argue is undesirable in absence of an unambiguous guideline): 

This seems unidiomatic. I believe a native user would say

"Your gain might be [set] too high."

So, if you change this line, then please make it sound more "round".